test/spec/protractor_sync_test.ts

```diff
@@ -193,6 +193,29 @@ describe('Protractor extensions', () => {
       el.scrollIntoView();
       expect(el.parent().scrollTop()).toEqual(500);
     }));
+
+    it('innerHTML', createTest(() => {
+      //Make sure we are starting on a fresh page
+      browserSync.get('data:,');
+
+      appendTestArea({
+        innerHtml: '<div class="div-with-text">Some Text In The Div</div>'
+      });
+
+      expect(elementSync.findVisible('.div-with-text').getInnerHtml()).toBe('Some Text In The Div');
+
+    }));
+
+    it('outerHTML', createTest(() => {
+      //Make sure we are starting on a fresh page
+      browserSync.get('data:,');
+
+      appendTestArea({
+        innerHtml: '<div class="div-with-text">Some Text In The Div</div>'
+      });
+
+      expect(elementSync.findVisible('.div-with-text').getOuterHtml()).toBe('<div class="div-with-text">Some Text In The Div</div>');
```

Comment:
Nice find Mindy! 
Minor:  Do we want to have the tests use this ``Some <b> HTML </b> In The Div``` to catch any inner HTML tags?  I'm fine if not.   
All looks good.  Approving.